src/StreamFactory.php

```diff
@@ -21,6 +21,7 @@
 
 class StreamFactory implements StreamFactoryInterface
 {
+
```

Comment:
I know this is just coding standard, but since it hasn't been reported by CI... based on the rest of the codebase it looks like this shouldn't be added.

Reply:
I'll check this out. Thanks for letting me know.

Reply:
Fixed!